From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <kleinschmidtorama@gmail.com>
Date: Tue, 24 Sep 2019 14:42:19 -0400
Subject: fix: GRIT blink strings for arm

Ensures that GRIT blink strings are generated for arm/arm64/woa
toolchains.

diff --git a/third_party/blink/public/strings/BUILD.gn b/third_party/blink/public/strings/BUILD.gn
index 83cfe650b7c13c81b87be1a2d3a5622e4bb1c9c9..bcbff5de44680ef2d23142148d16d28c382dda4d 100644
--- a/third_party/blink/public/strings/BUILD.gn
+++ b/third_party/blink/public/strings/BUILD.gn
@@ -3,8 +3,9 @@
 # found in the LICENSE file.
 
 import("//tools/grit/grit_rule.gni")
-
+strings_out_dir = "$root_gen_dir/third_party/blink/public/strings"
 grit("strings") {
+  output_dir = strings_out_dir
   source = "blink_strings.grd"
   outputs = [
     "grit/blink_strings.h",
